src/context/online.rs

```diff
@@ -140,15 +165,73 @@ impl AudioContext {
     /// the time at which the first sample in the buffer is actually processed
     /// by the audio output device.
     #[must_use]
+    #[allow(clippy::missing_panics_doc)]
     pub fn output_latency(&self) -> f64 {
-        self.backend.output_latency()
+        self.backend.lock().unwrap().output_latency()
```

Comment:
not sure of the thread layout here, but can't this lock() be a problem regarding the audio thread?

Reply:
that's something we might want to call periodically during rendering I guess

Reply:
I realize now the `backend` naming is confusing.
The `backend` object live in the control thread, so locking is no issue.
A better name would be `backend_manager`, because it passes instructions from the control thread (start, pause, close) and collects info from the actual backend in the render thread (latency, sink_id).
I will update the PR by renaming this struct